ublox/src/ubx_packets/packets.rs

```diff
@@ -1986,6 +1987,32 @@ struct EsfMeas {
 //     msss: u32,
 // }
 
+#[ubx_packet_recv]
+#[ubx(class = 0x10, id = 0x15, fixed_payload_len = 36)]
+struct EsfIns {
+    bit_field: u32,
```

Comment:
I think you can define a bitfield for this type as well

Reply:
Would version need to be skipped? (thought the bitflags only checks toggles at specific index)

Reply:
Looked like theres 8 bits of the version, maybe add extra version field, skip u16 as reserve, bitfield u8, and add version u8

Reply:
Yeah it looks like the bitfield stuff only supports flags that are one bit (makes sense). What you said would probably work - but I'm not sure we should break up a field like that. If the end goal is ubx->json we probably want to leave the fields "in tact". So maybe just skip this one for now